infra_overview/jenkins.rst

```diff
@@ -30,3 +30,14 @@ Gluster release managers and some developers have write access to selected
 parts of the Jenkins user interface through the jenkins-admins team on Github.
 Unless someone is a part of the Gluster CI team, this is the higest level of
 access they will be granted.
+
+Managing Jenkins jobs
+---------------------
+
+Jenkins jobs are managed with `Jenkins Job Builder`_. Editing existing jobs or
+adding new jobs are managed by submitting patches to the repository. A Jenkins
```

Comment:
So on first read, i wasn't sure where was the repository. The link do look like it point to jjb as a software more than the repository itself. Do you think this could be changed (e.g., a link on "repository" ?)